.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+line_length=120
```

Comment:
I'm using 120 from some time now. Beside that 8*4 = 32 - so there a 48 signs for 'for' and 'while' :P